portal-impl/src/com/liferay/portal/action/SessionTreeJSClickAction.java

```diff
@@ -74,6 +74,10 @@ else if (cmd.equals("layoutCheck")) {
 					boolean privateLayout = ParamUtil.getBoolean(
 						request, "privateLayout");
 
+					SessionTreeJSClicks.openLayoutNodes(
```

Comment:
If you do not have these changes, we will omit "0" as default layout id in our responses to ``checkedNodes``.

This causes synchronizing problem between ``localCheckedNodes`` and ``checkedNodes".

i.e ``localCheckedNodes`` will be ``0, 20138, 20167'' while `checkedNodes`` will be `` 20138, 20167''.

This will cause an issue at https://github.com/jonmak08/liferay-portal/pull/1611#issuecomment-245314352.